app/controllers/shared/api.php

```diff
@@ -311,6 +311,18 @@
                 }
                 break;
 
+            case 'token':
+                if (($auths['token'] ?? true) === false) {
+                    throw new Exception(Exception::USER_AUTH_METHOD_UNSUPPORTED, 'Token authentication is disabled for this project');
+                }
+                break;
```

Comment:
This needs to be added to the auth.php file in the config directory. Else it will break the `updateAuthStatus` endpoint

Reply:
Also what is "token" based auth? How does it sit well in this screen? 
![Screenshot 2024-01-16 at 1 35 49 AM](https://github.com/appwrite/appwrite/assets/20852629/f24b8b5d-d4d2-4c29-955b-8451abb2b910)

I am to understand that tokens are created by emailPassword, phone, magic URL etc. So if we add a new check box for `token` here, what does it mean to disable `token` based auth? 

does it mean, we also disable emailPassword, phone, magicURL etc? Because a token is generated by all those methods right?

Reply:
Good point, I think we should remove these. It doesn't fit in this screen. 

The user can already control where a token is created with the existed toggles.
If we let them disable token auth it would break magic url and phone auth.